### PR TITLE
Fix flow

### PR DESCRIPTION
**Pull request by xv2**

### CODE REVIEW

File: common/read-models/comments.js [hunk 1]
```diff
@@ -1,13 +1,20 @@
...
 
-const { COMMENT_CREATED } = events
+type CommentsState = Array<{
+  id: string,
+  text: string,
+  parentId: string,
+  storyId: string,
+  createdAt: number,
+  createdBy: string
+}>
```
Comment (timbset) [inline common/read-models/comments.js]: First. Extract inner type as a Comment. 
Second. Where is it used?

File: common/read-models/stories.js [hunk 2]
```diff
@@ -1,20 +1,33 @@
...
+  id: string,
+  type: 'ask' | 'show' | 'story',
+  title: string,
+  text: string,
+  link: string,
+  commentCount: number,
+  votes: Array<UserId>,
+  createdAt: number,
+  createdBy: UserId
+}>
```
Comment (timbset) [inline common/read-models/stories.js]: The same

File: common/read-models/users.js [hunk 3]
```diff
@@ -1,18 +1,23 @@
...
 import type { Event, UserCreated } from '../events'
-import events from '../events'
+import { USER_CREATED } from '../events'
 
-const { USER_CREATED } = events
+type UsersState = Array<{
+  id: string,
+  name: string,
+  createdAt: number
+}>
```
Comment (timbset) [inline common/read-models/users.js]: And here

File: flow-typed/events.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  userId: string,
+  text: string,
+  parentId: string,
+  commentId: string
+}>
+
+// User
+export type UserCreated = ResolveEvent<{
+  name: string
+}>
```
Comment (timbset) [inline flow-typed/events.js]: And in this file

**Review by timbset — CHANGES_REQUESTED** [review]

**Review by MrCheater — APPROVED** [review]

**Review by kilimondjaro — APPROVED** [review]

**Review by MrCheater — APPROVED** [review]

**Review by timbset — APPROVED** [review]

**Review by kilimondjaro — APPROVED** [review]

**Pull request merged**